pkl-core/src/main/java/org/pkl/core/ast/member/ObjectMember.java

```diff
@@ -35,9 +38,11 @@ public ObjectMember(
       SourceSection headerSection,
       int modifiers,
       @Nullable Identifier name,
-      String qualifiedName) {
+      String qualifiedName,
+      boolean isInIterable) {
```

Comment:
There's quite a lot of extra code throughout, all with a default `false` case. I don't care much, considering it's likely rather temporary, but might it be worth an extra constructor that defaults this to `false`?